coral-component-list/src/scripts/SelectList.js

```diff
@@ -221,10 +221,28 @@ class SelectList extends BaseComponent(HTMLElement) {
   set _tabTarget(value) {
     this.__tabTarget = value;
     
-    // Set all but the current set _tabTarget to not be a tab target:
-    this.items.getAll().forEach((item) => {
-      item.setAttribute('tabindex', item === value ? 0 : -1);
-    });
+    if(!this._groups){
+      // Set all but the current set _tabTarget to not be a tab target:
+      this.items.getAll().forEach((item) => {
+        item.setAttribute('tabindex', item === value ? 0 : -1);
+      });
+    }
+    else {
+      var groups=this._groups;
+      groups.getAll().forEach((group) => {
+        var firstChild=group.firstElementChild;
+        firstChild.setAttribute('tabindex', 0);
```

Comment:
I'm not sure that the headings for on option groups should ever receive keyboard focus. The important thing is that the descendant options within the groups receive keyboard focus, and that each group is labelled by its heading.

Reply:
This will select allow first option inside group to receive keyboard focus. Header will not be selected